src/anomalib/utils/callbacks/normalization/cdf_normalization.py

```diff
@@ -113,10 +113,11 @@ def _collect_stats(self, trainer: Trainer, pl_module: AnomalyModule) -> None:
          estimate the distribution of anomaly scores for normal data at the image and pixel level by computing
          the mean and standard deviations. A dictionary containing the computed statistics is stored in self.stats.
         """
-        _trainer = T.AnomalibTrainer(accelerator=trainer.accelerator, devices=trainer.num_devices, normalizer="none")
-        predictions = _trainer.predict(
+        _engine = engine.Engine(accelerator=trainer.accelerator, devices=trainer.num_devices, normalization="none")
```

Comment:
I know I'm annoying now, but wondering if we still get circular import if we import `Engine` from anomalib directly. `engine.Engine` look a bit peculiar

Reply:
I still get circular import error
```bash
Traceback (most recent call last):
  File "/home/ashwin/projects/anomalib_copy/tools/train.py", line 17, in <module>
    from anomalib import Engine
  File "/home/ashwin/projects/anomalib_copy/src/anomalib/__init__.py", line 8, in <module>
    from .engine import Engine
  File "/home/ashwin/projects/anomalib_copy/src/anomalib/engine/__init__.py", line 6, in <module>
    from .engine import Engine
  File "/home/ashwin/projects/anomalib_copy/src/anomalib/engine/engine.py", line 21, in <module>
    from anomalib.utils.callbacks.normalization import get_normalization_callback
  File "/home/ashwin/projects/anomalib_copy/src/anomalib/utils/callbacks/normalization/__init__.py", line 10, in <module>
    from .cdf_normalization import _CdfNormalizationCallback
  File "/home/ashwin/projects/anomalib_copy/src/anomalib/utils/callbacks/normalization/cdf_normalization.py", line 14, in <module>
    from anomalib import Engine
ImportError: cannot import name 'Engine' from partially initialized module 'anomalib' (most likely due to a circular import) (/home/ashwin/projects/anomalib_copy/src/anomalib/__init__.py)
```

Reply:
Alright, would it be ok if you add a comment there explaining why is it imported that way. In the future, it is highly likely to forget the reason :)

We could then merge this one I guess